pkgs/games/colobot/data.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchFromGitHub, cmake
```

```suggestion
{ stdenv, fetchFromGitHub, cmake
```

We don't need lib and stdenv.lib.